File: Space Station/Assets/DoorTriggerer.cs
using UnityEngine;
using System.Collections;

public class DoorTriggerer : MonoBehaviour {
	public GameObject door1;
	public GameObject door2;
	// Use this for initialization
	void Start () {
	
	}
	
	// Update is called once per frame
	void Update () {
	
	}

	void OnTriggerEnter(Collider col){
		Debug.Log("Trigger");
		if (col.gameObject.tag == "Door1") {
			Debug.Log("Door1");
			if(!door1.GetComponent<Animation>().isPlaying){
				door1.GetComponent<Animation>().Play();
			}
		}
	}
}

File: Space Station Non-Script/Assets/DoorTriggerer.js
#pragma strict

var airlockdoor : GameObject;
var securitydoor : GameObject;

function Start () {

}

function OnTriggerEnter (other : Collider) {
		if(other.gameObject.tag == "AirlockTrigger"){
				airlockdoor.GetComponent(SmoothMoveJS).Move();
		}
		if(other.gameObject.tag == "SecurityTrigger"){
				securitydoor.GetComponent(SmoothMoveJS).Move();
		}
	}

function Update () {

}